### ISSUE TITLE
[Bug] +, ÷, = Album Names Combine

### ISSUE THREAD

**Issue by echo-84** [SimpleMobileTools/Simple-Music-Player#446]
Music Files whose album names are +, ÷, = get combined into the + album acting as if they are all under the same album.

### Steps to Reproduce

1. Tag each music file with the proper Album name [+, ÷, =]
2. place in separate folders
3. move to phone
4. let app scan for Music

### Expected Behavior

- Each album is separated with their unique name and cover art

### Actual Behavior

- Each album is treated as one larger album using the first ones' cover art in this case +

### Additional Info
Retagging the music files using verbiage rather than symbol split the albums properly

**Comment by tibbi** [comment]
not sure what you mea, the albums name is literally "+"?

**Comment by echo-84** [comment]
Yes, they are [Ed Sheeran](https://musicbrainz.org/artist/b8a7c51f-362c-4dcb-a259-bc6e0095f0a6) albums whose names are literally the symbols +, ÷, and =

**Comment by naveensingh** [comment]
@tibbi this is a media store issue, I also have these albums, and the symbolic names are not properly handled by the media store. (These symbols also cause a native crash when using `MediaMetadataRetriever` on one Android 9 device).

We can fix it by grouping tracks into albums and artists manually (we already do this for any tracks not present in the media store). If we do that, we can address https://github.com/SimpleMobileTools/Simple-Music-Player/issues/351 too as that also requires manual grouping of tracks by `ALBUMARTIST` instead of `ARTIST` but there's a con:

https://github.com/SimpleMobileTools/Simple-Music-Player/blob/9a3e835f773b99a8b9e497e9c09798ad9968544e/app/src/main/kotlin/com/simplemobiletools/musicplayer/helpers/SimpleMediaScanner.kt#L86-L91

(**Workaround**: For anyone facing any similar issues, from version 5.17.0, you can move these files to **Audiobooks** folder in the storage root and they will be scanned properly)

**Comment by tibbi** [comment]
doesnt sound like a major issue, it isnt a priority now
